working/wildcards/feature-specification.md

```diff
@@ -127,8 +135,31 @@ means you can have multiple local declarations named `_` in the same namespace
 without a collision error. The initializer, if there is one, is still executed,
 but the value is not accessible.
 
-Other declarations: top-level variables, top-level function names, type names,
-member names, etc. are unchanged. They can be named `_` as they are today.
+### Record type positional fields
+
+```dart
+typedef R = (String _, String _);
+(int _, int _) record;
+```
+
+Record type field names will no longer emit an `INVALID_FIELD_NAME` warning.
+This will have no other effect since the names of positional record fields
+don't matter.
+
+### Local function declarations
+
+```dart
+void f() {
+  _() {} // Error.
+  _(); // Error.
+}
+```
+
+It's an error to declare a local function declaration named `_`.
```

Comment:
Good catch!

Reply:
That is what @lrhn suggested https://github.com/dart-lang/language/issues/3790#issuecomment-2104021741. We can do that. I'll make the change.